configs/mini_pupper_config/README.md

```diff
@@ -8,13 +8,12 @@ You don't need a physical robot to run the following demos.
 
 #### 1.1.1. Run the base driver:
 
-    ros2 launch mini_pupper_config bringup.launch.py
+    ros2 launch mini_pupper_config bringup.launch.py rviz:=true
```

Comment:
ros2 have a different way to supply args, 
sth like 
```
ros2 run package_name executable_name --ros-args -p param_name:=param_value


```
reference:  https://docs.ros.org/en/foxy/How-To-Guides/Node-arguments.html

Reply:
That is right about `ros2 run`, but in this case, we use `ros2 launch`.
`ros2 launch` can pass argument like `ros2 launch <package_name> <launch_file_name> background_r:=255`
ref: https://docs.ros.org/en/galactic/How-To-Guides/Launch-file-different-formats.html#setting-arguments